website/docs/misc/i18next.md

```diff
@@ -185,37 +218,78 @@ i18next handles plurals differently. It requires a separate key to be defined fo
 ```
 
 ```js
+import i18next from "i18next";
+
 i18next.t("key", { count: 0 }); // -> "items"
 i18next.t("key", { count: 1 }); // -> "item"
 i18next.t("key", { count: 5 }); // -> "items"
 ```
 
 ## Context
 
-By providing a context you can differ translations. Both i18next and Lingui have the context feature to differentiate messages.
+By providing context, you can differentiate translations for the same sentences or provide translators with more details. Both i18next and Lingui have the context feature to differentiate messages. However, Lingui provides an automatic additional "context" by including in the `.po` file the locations where each message is used - i18next can't do this from its plain JSON files.
 
-## Summary
+i18next sample:
+
+```js
+import i18next from "i18next";
+
+i18next.t("Right", { context: "direction" });
+```
+
+Lingui sample:
+
+```js
+import { msg } from "@lingui/core/macro";
+
+msg({
+  message: "Right",
+  context: "direction",
+});
+```
```

Comment:
Would it be usefull to add a `po` example here to show what was written in the description above: 

> Lingui provides an automatic additional "context" by including in the `.po` file the locations where each message is used - i18next can't do this from its plain JSON files.